### DISCUSSION TITLE
[BUG] Components, Directives and Pipes not listed by default in their module

### DISCUSSION THREAD

**Discussion by jllodra** [compodoc/compodoc#1055] [Ideas]
##### **Overview of the issue**

There are no Components / Directives / Pipes in the main menu, right below the Modules menu item.
If I open Modules, there are all the Modules and within every module, there is a menu item with Components and Directives and Pipes.
Is this normal behaviour? I would like to see all Components and Directives and Pipes independently of the Module they belong to.

![image](https://user-images.githubusercontent.com/273590/107045016-565bb100-67c5-11eb-8e51-43bc8963dadf.png)

The project is a library, with multiple modules, entry points, and uses ng-packgr

I was expecting this (notice Components and Directives menus)

![image](https://user-images.githubusercontent.com/273590/107046467-f49c4680-67c6-11eb-8790-7ea578fc50b5.png)

##### **Operating System, Node.js, npm, compodoc version(s)**

Win10, nodejs 14, compodoc 1.1.11

##### **Angular configuration, a `package.json` file in the root folder**

package.json is standard angular 10.x package.json generated by angular-cli

##### **Compodoc installed globally or locally ?**

Locally

##### **If possible sourcecode of the file where it breaks**

No break

##### **If possible your terminal logs before the error**

No errors, and this info is OK. It's more of a displaying issue I guess.

![image](https://user-images.githubusercontent.com/273590/107046159-a0916200-67c6-11eb-98fd-c8bb81332e96.png)

##### **Motivation for or Use Case**

I cannot see a comprehensive list of all Components, Directives, Pipes of the library.
I have to go through "Modules", which is visually a mess.
I would prefer not to see any Components / Directives submenus on Modules.

**Comment by ungureanuu** [comment]
Any updates regarding this? I have the same problem in a ecosystem of Angular11, nodejs 12.13 and compodoc installed globaly

**Comment by vogloblinsky** [comment]
Hi @jllodra 
the default behavior of compodoc is to display components inside their parent module. I convert this issue in a new discussion, and if there is many votes, i will try to add a flag for disabling this default behavior.
